src/test/kotlin/org/rustSlowTests/console/RsEvcxrTest.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rustSlowTests.console
+
+import org.rust.cargo.RsWithToolchainTestBase
+import org.rust.cargo.project.settings.toolchain
+import org.rust.cargo.toolchain.Cargo
+import java.io.PrintWriter
+import java.util.*
+
+class RsEvcxrTest : RsWithToolchainTestBase() {
+
+    fun testProcessOutputFormat() {
+        if (Cargo.checkNeedInstallEvcxr(project)) return
+
+        val process = createProcess()
+        val scanner = Scanner(process.inputStream)
+        val writer = PrintWriter(process.outputStream)
+
+        assertEquals(scanner.nextLine(), "Welcome to evcxr. For help, type :help")
+        expectPrompt(scanner, true)
+
+        // success command
+        writer.println("1 + 2")
+        writer.flush()
+        assertEquals(scanner.nextLine(), "3")
+        expectPrompt(scanner, true)
+
+        // fail command
+        writer.println("let foo = bar;")
+        writer.flush()
+        assert(scanner.nextLine().contains("not found in this scope"))
+        assert(scanner.nextLine().contains("cannot find value `bar` in this scope"))
+        expectPrompt(scanner, false)
+
+        process.destroy()
```

Comment:
Let's put it into `finally` block to destroy process even if some check above throws exception